src/@next/Select/selectStoryHelper/NonSearchableMultiSelect.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useCallback, useState } from 'react';
+import { Select, SelectProps } from '../Select';
+import { ActivatorSelectStyled } from './SelectStoryStyle';
+
+export const NonSearchableMultiSelect = ({
+  options,
+  disabled,
+  ...args
+}: SelectProps) => {
+  const [selected, setSelected] = useState([]);
+
+  const handleSelect = ({ value }: { value: string }) => {
+    if (selected.includes(value)) {
+      setSelected(selected.filter(option => option !== value));
+    } else {
+      setSelected([...selected, value]);
+    }
+  };
+
+  const removeTag = useCallback(
```

Comment:
why do we need useCallback here? 🤔

Reply:
looks like we don't really need to use here